mappings/net/minecraft/entity/passive/CatEntity.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_1451 net/minecraft/entity/passive/CatEntity
+	COMMENT Meow.
```

Comment:
Isn't this javadoc on the field and not the class? Or do I not understand the tiny format

Reply:
This is on the class since this is one level indent within the class. If this is doc on field it would be one level indent after the field.

Reply:
That is class javadoc.

This isn't tiny but enigma format

Reply:
Aye